tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -31,6 +31,8 @@ internal static class Resources
 
         public static string ErrorDetectingAgent(string url, string error) => $"Error while trying to reach agent at {url}: {error}";
 
+        public static string ConnectToEndpointFormat(string endpoint, string transport) => $"Trying to connect to agent at endpoint {endpoint} using {transport}";
```

Comment:
```suggestion
        public static string ErrorDetectingAgent(string url, string error) => $"Error connecting to Agent at {url}: {error}";

        public static string ConnectToEndpointFormat(string endpoint, string transport) => $"Connecting to Agent at endpoint {endpoint} using {transport}";
```

Reply:
There's a few more "agent" in lowercase above, too.

Reply:
I find it a bit weird to capitalize Agent, but done in https://github.com/DataDog/dd-trace-dotnet/pull/2337/commits/ed76a930c03fe7331b3bcbcb2b619abadfdb39c2